### PR TITLE
Fixed Zendesk End Chat Resolution bug

### PR DESCRIPTION
**Pull request by raj5036**
### What do you want to achieve?
- Fixed Zendesk End Chat Resolution bug

### PR Checklist

- [X] I have tested it locally and all functionalities are working fine.

### How was the code tested?

- Tested locally.

### CODE REVIEW

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 1]
```diff
@@ -7858,18 +7858,17 @@ var userOverride = {
                 '<div class="move-right mck-group-count-box mck-group-count-text ${displayGroupUserCountExpr}">${groupUserCountExpr}</div></div>' +
                 '<div class="blk-lg-12 mck-text-muted">${contLastSeenExpr}</div></div></div></div></a></li>';
             var csatModule =
-                '<div class="km-csat-skeleton"> <div class="mck-rated"> <span id="mck-resolved-text">' + 
-                MCK_LABELS['csat.rating'].CONVERSATION_RESOLVED + '</span><br><div id="separator"><span id="mck-rated-text">' +
-                MCK_LABELS['csat.rating'].CONVERSATION_RATED +
-                '</span><span class="mck-rating-container">{{html ratingSmileSVG}}</span></div></div><div class="mck-conversation-comment">${ratingComment}</div></div>';
+            '<div class="km-csat-skeleton"> <div class="mck-rated"> <span id="mck-resolved-text" class=${isConversationResolvedFromZendesk ? "" : "n-vis"}>' +
```
Comment (Prochnost55) [inline webplugin/js/app/mck-sidebox-1.0.js:7861]: this will not work in IE

File: webplugin/js/app/zendesk-chat-service.js [hunk 2]
```diff
@@ -257,6 +257,7 @@ function ZendeskChatService() {
             true
         );
         KommunicateUI.isConvJustResolved = true;
+        KommunicateUI.isConversationResolvedFromZendesk= true;
```
Comment (Prochnost55) [inline webplugin/js/app/zendesk-chat-service.js:260]: formating

File: webplugin/css/app/mck-sidebox-1.0.css [hunk 3]
```diff
@@ -5594,7 +5594,7 @@ div#mck-waiting-queue {
 	margin: 0 5px -4px 3px;
 }
 
-.km-csat-skeleton .mck-rated:after,
+/* .km-csat-skeleton .mck-rated:after,
```
Comment (Prochnost55) [inline webplugin/css/app/mck-sidebox-1.0.css:5597]: if not required we can remove this

**Review by Prochnost55 — CHANGES_REQUESTED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 4]
```diff
@@ -9050,10 +9050,16 @@ var userOverride = {
                                 '"' + userFeedback.comments.trim() + '"';
                         }
 
+                        var resolutionStatusClass = "";
+                        if (KommunicateUI.isConversationResolvedFromZendesk == false) {
```
Comment (gargshantnu) [inline webplugin/js/app/mck-sidebox-1.0.js:9054]:
```suggestion
                        if (!KommunicateUI.isConversationResolvedFromZendesk) {
```

**Review by gargshantnu — APPROVED** [review]

**Review by Prochnost55 — APPROVED** [review]

**Pull request merged**